measurements/attribute_based.py

```diff
@@ -62,15 +62,20 @@ def att_siz(dataloader, args):
             detect_info = {}
     elif FACE_DETECT == 1:
         cascPath = "util_files/haarcascade_frontalface_default.xml"
-        #faceCascade = cv2.CascadeClassifier(cv2.data.haarcascades + cascPath)
-        faceCascade = cv2.CascadeClassifier(cascPath)
+        try:
+            faceCascade = cv2.CascadeClassifier(cv2.data.haarcascades + cascPath)
+        except AttributeError:
+            faceCascade = cv2.CascadeClassifier(cascPath)
+        
     for i, (data, target) in enumerate(tqdm(dataloader)):
         attribute = target[1]
         # Only look at image if there is an attribute to analyze (Note attribute require a bbox around the person or thing to analyze)
         if len(attribute)> 1:
             shape = list(data.size())[1:]
             bboxes = attribute[1]
-            for bbox in bboxes:
+            for index in range(len(bboxes)):
+                bbox = bboxes[index]
```

Comment:
for better readability, maybe it's best to match notation and get rid of the bboxes variable and make this attribute[1][index]